docs/src/main/asciidoc/transaction.adoc

```diff
@@ -362,16 +362,21 @@ NOTE: The `event` object represents the transaction ID, and defines `toString()`
 TIP: In listener methods, you can access more information about the transaction in progress by accessing the `TransactionManager`,
 which is a CDI bean and can be ``@Inject``ed.
 
-== Configuring transaction log to be stored in a DataSource
+[[jdbcstore]]
+== Configure the transaction log to be stored in a dataSource
 
-The Narayana project has the capability to store the transaction logs into a JDBC Datasource; this should be our recommendation for users needing transaction recovery capabilities, especially when running in volatile containers.
+JDBCStore is a class in the Narayana transaction management library that facilitates the storage of transaction logs in a database by using JDBC.
 
-To enable this capability, you need to set `quarkus.transaction-manager.object-store.type` to `jdbc` explicitly. Also, you can specify a datasource name to be used for the transaction log storage by setting `quarkus.transaction-manager.object-store.datasource`. It will use the default datasource configuration if not specified.
+JDBCStore is our recommended approach for users needing transaction recovery capabilities, especially when running in volatile containers.
 
-If you enable `quarkus.transaction-manager.object-store.create-table`, the transaction log table will be created automatically if it does not exist.
+To enable this capability, explicitly set `quarkus.transaction-manager.object-store.type` to `jdbc`.
+You can also specify a datasource name for the transaction log storage by setting `quarkus.transaction-manager.object-store.datasource`.
+It will use the default datasource configuration if not specified.
 
-NOTE: When enabling this capability, the transaction node identifier must be set through `quarkus.transaction-manager.node-name`.
+When the `quarkus.transaction-manager.object-store.create-table` configuration is enabled, the transaction log table is automatically created if it does not already exist.
+However, to enable this capability, the transaction node identifier must be set through `quarkus.transaction-manager.node-name`.
```

Comment:
If the nodeId isn't set then a default value is used so this sentence is not clear - ie the transaction manager always has a nodeId (and it gets embedded in XIDs). Perhaps you are referring to requirement for a unique nodeId but if you are then the same statement will apply to any object store type and is not specific to the JDBCStore so it looks out of place to draw attention to it here.